### ISSUE TITLE
[meta] VoIP-related accessibility issues

### ISSUE THREAD

**Issue by anoadragon453** [vector-im/element-web#18516]
Originally mentioned by @t3chguy at https://github.com/matrix-org/matrix-react-sdk/pull/6520#issuecomment-892562292, this is meant to serve as a meta bug for various accessibility (a11y) issue that have been discovered surrounding the VoIP UI in Element web.

- [x] The floating buttons that appear over a user's avatar in a Matrix 1:1 call have no labels, so as a blind user I'd have no idea which one is the one that opens the dial pad, they're all just "button".
- [x] The call buttons vanish visually whilst the user's keyboard focus is on them (same for mouse focus).
- [x] The in-call dial pad disappears when hitting tab, making keyboard navigation difficult.
- [x] The backspace button in dial pad missing label "button"
![image](https://user-images.githubusercontent.com/2403652/128168566-9cb55e7f-2881-4e2d-8849-49c7df40ff5a.png)
- [x] The call button on the dial pad is missing any label, it's just "button"
![image](https://user-images.githubusercontent.com/2403652/128168726-1ac4aa2c-2364-425e-9a82-3104ac49e1a8.png)

**Comment by t3chguy** [comment]
the first point should already be resolved now

**Comment by Palid** [comment]
It's hard to triage it, but I'd say in terms of A11Y it's a major, as it makes it a lot harder for people with additional accessibility needs to use this feature.
